primitives/block/src/block.rs

```diff
@@ -232,8 +232,7 @@ impl Block {
         }
     }
 
-    /// Return the number of transactions in the block. If the block is a Macro
-    /// block it just returns zero, since Macro blocks don't contain any transactions.
```

Comment:
Why did we change it?

Reply:
I thought the function returns only the number of transactions of the block and there was no need to clarify the distinction between micro and macro

Reply:
I changed that actually in my commit, because as of this commit macro blocks now _do_ include transactions:
https://github.com/nimiq/core-rs-albatross/pull/1280/commits/26c8a848f500757d91f7452082d4636500d95e69#diff-aaa71b99350a00b080a331c7886311a206cfc2475c64d82748c711d9c4f10a04R242-R244

Reply:
Ah, but we didn't then update the primitives properly: https://github.com/nimiq/core-rs-albatross/blob/albatross/primitives/block/src/block.rs#L221